src/style/app.module.css

```diff
@@ -397,26 +404,31 @@
     :disabled,
     :checked
   ) {
-  background-color: transparent !important;
-  border: 1px solid var(--brown-color);
-  color: var(--dropdown-button-bg) !important;
-  border-color: var(--brown-color) !important;
+  box-shadow: 2.5px 2.5px 2.5px var(--dropdown-shadow);
+  border: 1px solid var(--dark-grey-border) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`